### ISSUE TITLE
Missing sin cos symbol for emboss

### ISSUE THREAD

**Issue by ydesgagn** [dyne/frei0r#14]
Hi,

I'm building the latest from master. When using with melt and kdenlive, when we use emboss, gdb report sin and cos symbols are missing. But I do see -lm in the makefile for emboss. Any idea on what can be wrong?

Thanks

**Comment by jaromil** [comment]
Does only gdb reports that or there is an error also out of the debugging environment?
If the error is reported only when in gdb then it may be due to the fact the `libm` binary has no debugging symbols.

**Comment by ydesgagn** [comment]
No, melt or kdenlive will seg fault if using the effect. That's why we had a look in gdb.

**Comment by jaromil** [comment]
`libm` is added here to all plugins https://github.com/dyne/frei0r/blob/master/src/CMakeLists.txt#L2
on my gnu/linux system emboss.so is linked to it after a cmake build
```
ldd emboss.so
	linux-vdso.so.1 (0x00007ffd01beb000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f4622605000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f462225a000)
	/lib64/ld-linux-x86-64.so.2 (0x00005632e40e5000)
```

**Comment by ydesgagn** [comment]
Sorry for the delay but it took us while to understand. When building:

Set CFLAGS=. In such case emboss crash.

If we use the same commands but unset CFLAGS instead emboss will work fine.

Any clue?

**Comment by jaromil** [comment]
Hi! was this problem solved at last? Noone else reported so I wonder if this was due to a particular build setup.

**Comment by ydesgagn** [comment]
I'm still using the workaround to make it work. We build this once a month. Without the unset CFLAGS we still have the crash.

**Comment by jaromil** [comment]
OK! Are you using configure/make or cmake/make to build frei0r this way?

**Comment by jaromil** [comment]
I think this fixes it, thanks for hanging out here 7011cd4d8a2b1c4627585cde6c8d7a2f5b6315eb
